tools/integration_tests/implicitdir/implicitdir_test.go

```diff
@@ -62,15 +62,17 @@ func setUpTestDir() error {
 	return nil
 }
 
-func mountGcsfuse() error {
+func mountGcsfuse(flag string) error {
 	mountCmd := exec.Command(
 		binFile,
 		"--implicit-dirs",
 		"--debug_gcs",
 		"--debug_fs",
 		"--debug_fuse",
+		"--experimental-enable-storage-client-library=false",
```

Comment:
why do you have this flag here? For the test with flag: "--experimental-enable-storage-client-library=true - are you passing it the same flag twice?

Reply:
You told me that we should have default flags and also we are passing it as params for all possible values.